### PR TITLE
LPS-170720 Disabling cache for every headless request

### PR DESCRIPTION
**Pull request by 4lejandrito**
Fixes https://issues.liferay.com/browse/LPS-170720.

### CODE REVIEW

**Comment by 4lejandrito** [comment]
ci:test:sf

**Comment by 4lejandrito** [comment]
ci:test:relevant

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/container/response/filter/CacheContainerResponseFilter.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.Provider;
+
+/**
+ * @author Alejandro Tardín
+ */
+@Priority(Priorities.USER + 10)
```
Comment (ccorreagg) [inline modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/container/response/filter/CacheContainerResponseFilter.java:31]: Is this priority necessary?

Comment (4lejandrito) [reply modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/container/response/filter/CacheContainerResponseFilter.java]: No 😓 .

**Comment by 4lejandrito** [comment]
ci:forward

**Review by LuismiBarcos — APPROVED** [review]

**Review by ccorreagg — APPROVED** [review]
LGTM now!

**Pull request closed without merging**